client/notifications/notification.js

```diff
@@ -42,10 +43,14 @@ Meteor.startup(function() {
 						KonchatNotification.newMessage(notification.payload.rid);
```

Comment:
I'm not clear on why embeds only fire notifications when the room is focused, instead of unfocused. I'm 99% sure this is an existing bug, but I've left it as is for now.  (L40-41 above)